### PR TITLE
fix(atomic): specify use of connectedCallback for resultContext method

### PR DESCRIPTION
**Pull request by ThibodeauJF**
https://coveord.atlassian.net/browse/KIT-1277

### CODE REVIEW

File: packages/atomic/src/components/result-template-components/result-template-decorators.tsx [hunk 1]
```diff
@@ -68,21 +68,20 @@ const resultContextEventName = 'atomic/resolveResult';
  *
  * This method is useful for building custom result template elements, see [Create a Result List](https://docs.coveo.com/en/atomic/latest/usage/create-a-result-list/) for more information.
  *
+ * You should use the method in the [connectedCallback lifecycle method](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements#using_the_lifecycle_callbacks).
```
Comment (jpmarceau) [inline packages/atomic/src/components/result-template-components/result-template-decorators.tsx:71]:
```suggestion
 * You should use the method in the [`connectedCallback` lifecycle method](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements#using_the_lifecycle_callbacks).
```

**Review by jpmarceau — APPROVED** [review]
👍

File: packages/atomic/src/pages/examples/custom.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
     <script>
       (async () => {
         await customElements.whenDefined('atomic-search-interface');
-        const searchInterface = document.querySelector('#search');
```
Comment (samisayegh) [inline packages/atomic/src/pages/examples/custom.html:14]: Is the `#search` id still needed? Or can it be removed from the html element?

**Review by samisayegh — APPROVED** [review]

**Review by npushkarskii — APPROVED** [review]

File: packages/atomic/src/pages/examples/custom.html (hunk 2 above)
Comment (ThibodeauJF) [reply packages/atomic/src/pages/examples/custom.html:14]: Not needed & it wasn't on the html element 😢

**Pull request merged**